strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/io/MavenMetadataExpirationSingleGroupCaseTest.java

```diff
@@ -42,52 +47,51 @@
     @Inject
     private GroupRepositoryProvider groupRepositoryProvider;
 
-
-    @BeforeEach
-    public void initialize(TestInfo testInfo)
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class })
+    @Test
+    public void groupRepositoryVersionLevelMetadataShouldBeRefreshedAsItsSingleProxySubrepository(@TestRepository(storage = STORAGE0,
+                                                                                                                  repository = REPOSITORY_LOCAL_SOURCE,
+                                                                                                                  layout = Maven2LayoutProvider.ALIAS,
+                                                                                                                  policy = RepositoryPolicyEnum.SNAPSHOT)
+                                                                                                  Repository localRepository,
+                                                                                                  @TestRepository(storage = STORAGE0,
+                                                                                                                  repository = REPOSITORY_HOSTED + "-groupRepositoryVersionLevelMetadataShouldBeRefreshedAsItsSingleProxySubrepository",
```

Comment:
Reason for adding suffix to repository is because inside the test we have `mockResolvingProxiedRemoteArtifactsToHostedRepository` metod which when called calls one other method `mockResolvingProxiedRemoteArtifactToHostedRepository` inside class `BaseMavenMetadataExpirationTest`. In this method the reository name is resolved using `getRepositoryName` method which expects a TestInfo argument and resolves the repository name as :
```
String methodName = testInfo.getTestMethod().get().getName();
        return repositoryId + "-" + methodName;

```
Therefore when I was not passing the suffix with repository, it was throwing `NullPointerException`

Reply:
@yugander-krishan-singh / @fuss86 : Do we still need the `TestInfo` here?

Reply:
@yugander-krishan-singh : If we really must use the suffixes, could you please abbreviate them to just single letters per word, as per our convention?

Reply:
@carlspring the `testInfo` argument appends the method name as suffix. Therefore I am also appending the "-METHOD_NAME". The test will fail if suffix doesn't match the test name.

Reply:
I see, thanks for the explanation.